bindings/netstandard/ElectionGuard/ElectionGuard.ElectionSetup/Guardian.cs

```diff
@@ -107,6 +108,80 @@ public class Guardian : DisposableBase
     public CeremonyDetails CeremonyDetails { get; set; }
     public Dictionary<string, ElectionPartialKeyBackup> BackupsToShare { get; set; } = new();
 
+    /// <summary>
+    /// Initialize a guardian with the specified arguments.
+    /// </summary>
+    public Guardian(
+        string guardianId,
+        ulong sequenceOrder,
+        int numberOfGuardians,
+        int quorum,
+        string keyCeremonyId)
+    {
+        GuardianId = guardianId;
+        SequenceOrder = sequenceOrder;
+
+        _electionKeys = new(guardianId, sequenceOrder, quorum);
+        CeremonyDetails = new(keyCeremonyId, numberOfGuardians, quorum);
+
+        SaveGuardianKey(_electionKeys.Share());

```

Comment:
again, correct by construction dictates that we save our own guardian key to the collection to remain compliant with the V2 process.